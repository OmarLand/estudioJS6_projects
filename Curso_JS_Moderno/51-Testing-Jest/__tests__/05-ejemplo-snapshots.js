const cliente = {
    nombre  : 'Omar Jesus',
    balance : 500,
    tipo    : 'Premium'
};

describe('Testing al cliente', () => {
    test('Es Omar Jesus', () => {
        expect( cliente.nombre ).toMatchSnapshot()
    })
});