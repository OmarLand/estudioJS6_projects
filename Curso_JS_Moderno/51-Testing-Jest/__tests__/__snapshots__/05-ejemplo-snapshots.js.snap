// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Omar Jesus 1`] = `"Omar Jesus"`;
